oaknational/Oak-Web-Application
# fix(performance): summary image loading should be prioritised

## Description

- converts plan-lesson svg to png (to save space 255kb -> 34kb)
- prioritise loading of summary card image

## How to test

1. Go to {cloud link}
2. go to plan a lesson
3. The image should load quickly

## Checklist
